src/test/java/com/emc/object/s3/S3MetadataSearchTest.java

```diff
@@ -128,7 +128,7 @@ public void testQueryObjects() throws Exception {
         QueryObjectsResult result = client.queryObjects(request);
         Assert.assertFalse(result.isTruncated());
         Assert.assertEquals(bucketName, result.getBucketName());
-        Assert.assertEquals("NO MORE PAGES", result.getNextMarker());
+        Assert.assertEquals("null", result.getNextMarker());
```

Comment:
1) is the response really setting "null" as a string value?  seems like it should just omit the element
2) this should check for both possibilities, otherwise this test will fail for older ECS versions (<3.4)